### PR TITLE
Fixed build procedure in README.

### PR DESCRIPTION
**Pull request by draghuram**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,9 +33,8 @@ We encourage pull requests and issues tracking via Github, and the [target-devel
 ##### Building tcmu-runner
 
 1. Clone this repo.
-1. Type `./install_dep.sh` to install development packages for dependencies, or you can do it manually:
+1. Type `./extra//install_dep.sh` to install development packages for dependencies, or you can do it manually:
```
Comment (mikechristie) [inline README.md]: Thanks for the fixup. I am not very familiar with markdown. Is the extra "/" needed between directories?

Comment (draghuram) [reply README.md]: No, it was a typo. I will fix it. Thanks for the catch.

**Comment by draghuram** [comment]
I fixed the typo and pushed the change.

**Comment by mikechristie** [comment]
Thanks.

**Pull request merged**